physionet-django/physionet/settings/base.py

```diff
@@ -107,6 +113,15 @@
 # Session management
 
 SESSION_COOKIE_SECURE = True
+SESSION_EXPIRE_AT_BROWSER_CLOSE = True
+
+# Idle timeout
+SESSION_SECURITY_WARN_AFTER = config('SESSION_SECURITY_WARN_AFTER', cast=int, default=540)
+SESSION_SECURITY_EXPIRE_AFTER = config('SESSION_SECURITY_EXPIRE_AFTER', cast=int, default=600)
+
+# Absolute timeout
+SESSION_COOKIE_AGE = config('SESSION_COOKIE_AGE', cast=int, default=43200)
```

Comment:
Do we need this as a setting? I think it's already supported: https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-SESSION_COOKIE_AGE